### DISCUSSION TITLE
WearOS component Dexcom One+ sensor - watch as collector

### DISCUSSION THREAD

**Discussion by PhilTU** [NightscoutFoundation/xDrip#3541] [Q&A]
I have a Dexcom One+ sensor running well in xDrip+ and I switched it to another phone which has WearOS watch (pixel) and xDrip+. The watch and phone work well together, but the "watch as collector" does not.

I realised that the watch can be in range of the phone but the sensor can be out of range, and in this scenario local collection by the watch doesn't happen. I tried force collection too. Today I also switched off the phone and still nothing.

I realise the One+ is a newish sensor and that WearOS xDrip+ lacks an active developer, but should I be expecting this to work ?

**Comment by Navid200** [comment]
Direct collection from WearOS is quite limited.

I didn't think it worked with Dex One, G7 or One+.

**Reply by PhilTU** [reply]
Seems to match my experience. Thanks,
